build_ios/Pods/Target Support Files/Pods-Play!/GeneralSettings.xcconfig

```diff
@@ -33,9 +33,9 @@ GCC_WARN_UNDECLARED_SELECTOR = YES
 GCC_SYMBOLS_PRIVATE_EXTERN = YES
 GCC_INPUT_FILETYPE = sourcecode.cpp.objcpp
 
-FRAMEWORK_PATH = ../../Framework/
-CODEGEN_PATH = ../../CodeGen/
-BOOST_FRAMEWORK_PATH = ../../Dependencies/BoostMac/ios/framework/
+FRAMEWORK_PATH = ${PODS_ROOT}/../../../Framework/
+CODEGEN_PATH = ${PODS_ROOT}/../../../CodeGen/
+BOOST_FRAMEWORK_PATH = ${PODS_ROOT}/../../../boost_1_58_0
```

Comment:
../boost_1_58_0 should be ../Dependencies/BoostMac/ios/framework/ in the submission version of this commit